### ISSUE TITLE
Error after update to 0.4.5

### ISSUE THREAD

**Issue by EmanueleHB** [TonicArtos/SuperSLiM#59]
After updating to 0.4.5 i get this error when trying to display the list.

```java
java.lang.NullPointerException: Attempt to read from field 'int com.tonicartos.superslim.SectionData.firstPosition' on a null object reference
```

**Comment by TonicArtos** [comment]
Do you have the line number from the trace?

**Comment by EmanueleHB** [comment]
Uhm, strange. I can't reproduce it now. There was probably something wrong with my code then.

**Comment by TonicArtos** [comment]
No problems :) I'm not too worried about it due to the changes happening in v0.5. However, if you do run into it again, I'll fix it on the 0.4 branch.

**Comment by EmanueleHB** [comment]
If it happens I'll let you know :)Thanks
